docs/package-overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview of PHP-HTTP packages
+
+There are many strategies how adapters and other components can be installed. However they are the same in one thing: they can be installed via [Composer](http://getcomposer.org/).
+
+## httplug
+
+**Interfaces for HTTP Client**
+
+- Namespace: `Http\Client`
+- Repository: https://github.com/php-http/httplug
+- Documentation: (Httplug)[httplug.md]
+
+## message-factory
+
+**Abstraction to create PSR-7 requests without binding to a specific implementation**
+ 
+- Namespace: `Http\Message`
+- Repository: https://github.com/php-http/message-factory
+- Documentation: [Message Factory](message-factory.md) 
+
+## *-adapter
```

Comment:
I think we could use both. The basic is the client, since it is a client interface. The next level is an adapter (which is still a client).